### ISSUE TITLE
Add docs on fork

### ISSUE THREAD

**Issue by jtagcat** [charles-haynes/whatapi#4]
- What does it change-improve on?
 - Why was it forked?
 - Is it `-ng` (rewrite/completely different), does it replace (and abandon) kvdh/whatapi?

**Comment by cehbz** [comment]
Yeah, that's a good suggestion. I'll add a `readme.md` to the todo list. Short answer is I wanted it to be idiomatic Go.
